@xen-orchestra/xapi/host.mjs

```diff
@@ -31,7 +33,21 @@ class Host {
    *
    * @param {string} ref - Opaque reference of the host
    */
-  async smartReboot($defer, ref) {
+  async smartReboot($defer, ref, bypassBlockedSuspend = false) {
+    const residentVmRefs = await this.getField('host', ref, 'resident_VMs')
+    const vmsWithSuspendBlocked = await asyncMap(residentVmRefs, ref => this.getRecord('VM', ref)).filter(
```

Comment:
What about the control domain VM and the current VM (when XOA is on the host being rebooted)?

Reply:
Only the [current VM](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/xapi/host.mjs#L53) seem to escape the `suspend` operation. I will fix that.

Reply:
Both VMs are excluded from the "suspend" operation. I will fix it.